components/Flipper/FlipRestrictionList/EditRestriction/EditRestriction.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import { Button } from 'react-bootstrap'
+import ItemFilter from '../../../ItemFilter/ItemFilter'
+import Tooltip from '../../../Tooltip/Tooltip'
+
+interface Props {
+    newRestriction: FlipRestriction
+    onSearchResultClick(item: SearchResultItem)
+    filters: FilterOptions[]
+    onFilterChange(filter?: ItemFilter)
+    addEditedFilter(overrideExisting?: boolean)
+    onNewRestrictionCancel()
+    overrideEditedFilter()
+}
+
+function EditRestriction(props: Props) {
+    let getButtonVariant = (range: string): string => {
+        return range === props.newRestriction.type ? 'primary' : 'secondary'
+    }
+
+    return (
+        <div>
+            <ItemFilter
+                defaultFilter={props.newRestriction.itemFilter}
+                filters={props.filters}
+                forceOpen={true}
+                onFilterChange={props.onFilterChange}
+                ignoreURL={true}
+                autoSelect={false}
+            />
+            <span>
+                <Tooltip
+                    type="hover"
+                    content={
+                        <Button
+                            variant="success"
+                            onClick={() => {
+                                props.addEditedFilter(false)
+                            }}
+                        >
+                            Add
+                        </Button>
+                    }
+                    tooltipContent={
+                        <span>
+                            Adds the filter to the selected restriction(s). If a specific filter is already present, it is <b>not</b> overwritten.
+                        </span>
+                    }
+                />
+                <Tooltip
+                    type="hover"
+                    content={
+                        <Button
+                            variant="success"
+                            onClick={() => {
+                                props.addEditedFilter(true)
+                            }}
+                            style={{ marginLeft: '20px' }}
+                        >
+                            Add (override existing)
+                        </Button>
+                    }
+                    tooltipContent={
+                        <span>
+                            Adds the filter to the selected restriction(s). If a specific filter is already present, it <b>is overwritten</b>.
+                        </span>
+                    }
```

Comment:
same for this.
also when removing a filter the add can be clicked but does nothing which I found to be pretty frustrating.  
Maybe make it live preview the changes after the update to avoid confusion